src/main/java/org/sagebionetworks/web/client/widget/entity/file/FileTitleBar.java

```diff
@@ -105,15 +107,14 @@ public void configure(EntityBundle bundle, ActionMenuWidget actionMenu) {
 	}
 
 	public void getLatestVersion() {
-		// determine if we should show report the version (only shows if we're looking at an older version
-		// of the file).
+		// determine if we should report the version as "Current"
 		jsClient.getEntityVersions(entityBundle.getEntity().getId(), 0, 1, new AsyncCallback<List<VersionInfo>>() {
 			@Override
 			public void onSuccess(List<VersionInfo> results) {
 				if (!results.isEmpty()) {
 					Long currentVersionNumber = results.get(0).getVersionNumber();
 					Long viewingVersionNumber = ((FileEntity) entityBundle.getEntity()).getVersionNumber();
-					view.setVersionUIVisible(!currentVersionNumber.equals(viewingVersionNumber));
+					view.setVersionUICurrentVisible(currentVersionNumber.equals(viewingVersionNumber));
```

Comment:
The version number is now always shown for Files, so we're just toggling whether or not we show (Current)